configure

```diff
@@ -25745,8 +25751,8 @@ for module in $ac_static_modules ; do
         as_fn_error $? "specified archive '$thearch' does not match expected module archive name '$moduledir.a' -- aborting" "$LINENO" 5
       fi
 
-      ac_static_modules=`echo "$ac_static_modules" | sed -e "s/$module//g"`
-      ac_build_static_modules=`echo "$ac_build_static_modules" | sed -e "s/modules\/$module//g"`
+      ac_static_modules=`echo "$ac_static_modules" | sed -e "s/$modulename\.o//g"`
+      ac_build_static_modules=`echo "$ac_build_static_modules" | sed -e "s/modules\/$modulename\.o//g"`
```

Comment:
This build bug is very specifically located here.   Why?  It's the combination of the `.` wildcard character, _and_ the use of the `/g` modifier, which says to remove _all_ matching occurrences, not just the first matching occurrence.

While I could fix this issue simply by removing that `g` modifier, I feel that that would still leave the subtle `.` matching in place, which would arise to cause surprising issues later.